### DISCUSSION TITLE
ui / open do the same thing?

### DISCUSSION THREAD

**Discussion by melMass** [vitest-dev/vitest#6888] [Feedback]
Hi,

I just switched to vitest from jest, it's a real joy in comparaison! 
I have a few minor issues I wanted to be sure about:
- `ui` and `open` in options seems to do the same thing, as in `open` enables `ui`, and only specifying `ui` still `opens`...
- `clearScreen` both in my vite.config.js or from the CLI isn't clearing my screen, reading the docs it should be the default, my only lead currently is that as part of the monorepo I ran these tests on a few package do define `clearScreen` false (tauri apps as it conflicts with tauri's reporting)

**Comment by sheremet-va** [comment]
`open` is true by default when `ui` is enabled

if you manually enable `open`, it enables `ui`, but you can do `ui: true, open: false`
